### PR TITLE
Enable JFR modules when JFR is enabled

### PR DESCRIPTION
**Pull request by thallium**

### CODE REVIEW

**Comment by thallium** [comment]
@tajila FYI

File: closed/custom/common/Modules.gmk [hunk 1]
```diff
@@ -41,15 +41,20 @@ else
 	#
 endif
 
+ifeq (false,$(OPENJ9_ENABLE_JFR))
+	MODULES_FILTER += \
+		jdk.jfr \
+		jdk.management.jfr \
+		#
```
Comment (keithc-ca) [inline closed/custom/common/Modules.gmk:48]: Outside a rule, a non-continuation line cannot begin with a tab. Please follow the pattern of lines 45-48:
```
  MODULES_FILTER += \
	jdk.jfr \
	jdk.management.jfr \
	#
```

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile amac jdknext

**Comment by keithc-ca** [comment]
Build aborted: changes in master branch of openj9 require changes that are only in the openj9-staging branch of this repository.
Local builds (both with and without JFR) succeeded with this change applied to the openj9-staging branch.

**Comment by tajila** [comment]
@thallium please add JDK11, 17, 21, and 23 versions

**Comment by keithc-ca** [comment]
> @thallium please add JDK11, 17, 21, and 23 versions

Won't that require the metadata files be added like in #833?

**Comment by tajila** [comment]
> Won't that require the metadata files be added like in https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/833?

Yes, I've already requested those files to be added.

**Comment by keithc-ca** [comment]
@thallium Please confirm that the following are back-ports of this change:
* https://github.com/ibmruntimes/openj9-openjdk-jdk23/pull/41
* https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/194
* https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/381
* https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/807

In the future, it would be helpful if you could make such connections as you create pull requests.

**Comment by thallium** [comment]
Thanks. Yes they are back-ports of this change.

**Pull request merged**